airbyte-integrations/connectors/source-gcs/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM nickgryg/alpine-pandas:3.9.12 as base
```

Comment:
Could you please try adding `RUN apt-get install lib32stdc++6` to Dockerfile instead of using a custom base image? (and adding pandas to setup.py)

Reply:
I tried adding this: `apk --no-cache add libstdc++`, `apk --no-cache add libstdc++6` and `lib32stdc++6`  
but got the following error still: 
```
Traceback (most recent call last):
  File "/airbyte/integration_code/main.py", line 9, in <module>
    from source_gcs import SourceGCS
  File "/airbyte/integration_code/source_gcs/__init__.py", line 6, in <module>
    from .source import SourceGCS
  File "/airbyte/integration_code/source_gcs/source.py", line 23, in <module>
    from .helpers import construct_file_schema, get_gcs_blobs, read_csv_file, get_stream_name
  File "/airbyte/integration_code/source_gcs/helpers.py", line 8, in <module>
    import pandas as pd
  File "/usr/local/lib/python3.9/site-packages/pandas/__init__.py", line 48, in <module>
    from pandas.core.api import (
  File "/usr/local/lib/python3.9/site-packages/pandas/core/api.py", line 47, in <module>
    from pandas.core.groupby import (
  File "/usr/local/lib/python3.9/site-packages/pandas/core/groupby/__init__.py", line 1, in <module>
    from pandas.core.groupby.generic import (
  File "/usr/local/lib/python3.9/site-packages/pandas/core/groupby/generic.py", line 76, in <module>
    from pandas.core.frame import DataFrame
  File "/usr/local/lib/python3.9/site-packages/pandas/core/frame.py", line 172, in <module>
    from pandas.core.generic import NDFrame
  File "/usr/local/lib/python3.9/site-packages/pandas/core/generic.py", line 169, in <module>
    from pandas.core.window import (
  File "/usr/local/lib/python3.9/site-packages/pandas/core/window/__init__.py", line 1, in <module>
    from pandas.core.window.ewm import (
  File "/usr/local/lib/python3.9/site-packages/pandas/core/window/ewm.py", line 15, in <module>
    import pandas._libs.window.aggregations as window_aggregations
ImportError: Error loading shared library libstdc++.so.6: No such file or directory (needed by /usr/local/lib/python3.9/site-packages/pandas/_libs/window/aggregations.cpython-39-aarch64-linux-gnu.so)
```

Reply:
I have found a solution! Please use `slim` image instead of `alpine`, just like [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-salesforce/Dockerfile#L1)